sceptre/scipool/templates/sc-tag-options.j2

```diff
@@ -22,7 +22,8 @@ Resources:
   {% endfor %}
 {% endif %}
 {% if sceptre_user_data.CostCenters is defined %}
-  {% for CostCenter in sceptre_user_data.CostCenters %}
+  {% set CostCenters = sceptre_user_data.CostCenters | fromjson %}
```

Comment:
Have you tested this locally? I'm not finding much documentation for `jinja2_from_json_extension`, but it looks like maybe this should be `from_json` instead of `fromjson`? https://github.com/metwork-framework/jinja2_from_json_extension/blob/v1.0.0/jinja2_from_json_extension/__init__.py#L20

Reply:
you are correct, the jinja extension package does not contain any docs.  I had to look at the code to figure out how it works.  You are also correct that it should be `from_json`, I have updated to the correct filter name.  I have tested the latest update of this PR and it does seem to work as expected.